binr/r2pm/r2pm

```diff
@@ -330,16 +333,24 @@ R2PM_GIT() {
 	cd "${R2PM_GITDIR}"
 	if [ -d "${DIR}" ]; then
 		cd "${DIR}"
-		git checkout master
-		git pull
+		if [ "${R2PM_GITBRANCH}" = "master" ]; then
```

Comment:
uhm... some repos are not using main and other non-master branch for their master branches..

Reply:
Really, all I want is a way to disable the checkout/pull of the current GITDIR, so it builds whatever is currently in the folder for testing.  Can I juts make it a boolean env var and just make it skip the checkout/pull?